UnicornEngine/video/source/vulkan/Image.cpp

```diff
@@ -14,85 +15,85 @@ namespace video
 namespace vulkan
 {
 Image::Image(vk::PhysicalDevice physicalDevice,
-    vk::Device device,
-    vk::Format format,
-    vk::ImageUsageFlags usage,
-    uint32_t width,
-    uint32_t height): m_device(device)
-                      , m_image(nullptr)
-                      , m_deviceMemory(nullptr)
-                      , m_format(format)
-                      , m_width(width)
-                      , m_height(height)
-                      , m_initialized(false)
+             vk::Device device,
```

Comment:
Please do not add extra indents to align values. One extra level for each scope is usually enough.

```c++
Image::Image(vk::PhysicalDevice physicalDevice,
    vk::Device device,
    vk::Format format,
    vk::ImageUsageFlags usage,
    uint32_t width,
    uint32_t height
) : m_device( device )
    , m_image( nullptr )
    , m_deviceMemory( nullptr )
    , m_format( format )
    , m_width( width )
    , m_height( height )
    , m_initialized( false )
```

In previous cases it didn't making it hard to read, but here it did